apps/emqx/src/emqx_passwd.erl

```diff
@@ -141,4 +141,4 @@ pbkdf2(MacFun, Password, Salt, Iterations, DKLength) ->
     pbkdf2:pbkdf2(MacFun, Password, Salt, Iterations, DKLength).
 
 hex(X) when is_binary(X) ->
-    pbkdf2:to_hex(X).
+    binary:encode_hex(X, lowercase).
```

Comment:
Q: This one though arrived in OTP 26 according to documentation, are we OK with this?

Reply:
oh, thanks. i'll use string lowercase.